### PR TITLE
Toast fallback toastr

### PR DESCRIPTION
**Pull request by nschonni**
For #605
In browsers that don't support webkitNotifications, it adds a peek pop-up inside the JabbR window using [toastr](https://github.com/CodeSeven/toastr)

Rebased version of #608

### CODE REVIEW

File: JabbR/Chat.css [hunk 1]
```diff
@@ -104,7 +104,6 @@
   height: 20px;
   left: 55px;
   position: absolute;
-  display: none;
```
Comment (davidfowl) [inline JabbR/Chat.css:107]: why?

Comment (nschonni) [inline JabbR/Chat.css:107]: Because now we don't have to hide the button, all browser now (kinda) support the notifications

Comment (samandmoore) [inline JabbR/Chat.css:107]: so then are we saying that in-browser-window toast feature === desktop-notification-toast?

Comment (nschonni) [inline JabbR/Chat.css:107]: no, just a fallback. you can try it out http://jabbr-nick.apphb.com/#/rooms/toast

**Comment by davidfowl** [comment]
Can you rebase and update against latest?

**Comment by nschonni** [comment]
I already rebased it, what's confilicting?

**Comment by davidfowl** [comment]
Overall this isn't useful because the notifications only live within jabbr itself.

**Pull request closed without merging**